lib/vagrant-orchestrate/command/push.rb

```diff
@@ -56,7 +56,8 @@ def execute
           argv = parse_options(opts)
           return unless argv
 
-          guard_clean unless ENV["VAGRANT_ORCHESTRATE_NO_GUARD_CLEAN"]
+          guard_clean unless ENV["VAGRANT_ORCHESTRATE_NO_GUARD_CLEAN"] && \
+                             @env.vagrantfile.config.orchestrate.disable_commit_guard
```

Comment:
I'll double check but I tested this and believe it is right because of the unless


> On Aug 13, 2015, at 8:04 PM, Ryan Breen <notifications@github.com> wrote:
> 
> In lib/vagrant-orchestrate/command/push.rb:
> 
> > @@ -56,7 +56,8 @@ def execute
> >            argv = parse_options(opts)
> >            return unless argv
> >  
> > -          guard_clean unless ENV["VAGRANT_ORCHESTRATE_NO_GUARD_CLEAN"]
> > +          guard_clean unless ENV["VAGRANT_ORCHESTRATE_NO_GUARD_CLEAN"] && \
> > +                             @env.vagrantfile.config.orchestrate.disable_commit_guard
> Should it be || instead of &&?
> 
> —
> Reply to this email directly or view it on GitHub.
>